### ISSUE TITLE
The first action isn't focused automatically on Actions Panel

### ISSUE THREAD

**Issue by v-xuanzh** [microsoft/AzureStorageExplorer#1599]
**Storage Explorer Version**: 1.9.0
**Branch:** master
**Build:** 20190729.3
**Platform/OS:** Linux Ubuntu 19.04/Windows 10/macOS High Sierra
**Architecture:** ia32/x64
**Regression From:** Previous release 1.9.0

**Steps to reproduce:**
1. Click one storage account -> Focus on the 'Actions' tab on 'Actions/Properties' Panel.
2. Observe whether the first action is focused or not.

**Expected Experience:**

The first action is focused.

**Actual Experience:**

The first action isn't focused.
![image](https://user-images.githubusercontent.com/34729022/62116497-d9589d00-b2ec-11e9-99fd-626fcc4027fa.png)

**Comment by craxal** [comment]
@v-xuanzh How exactly did you focus the “Actions” tab? Which element _does_ have focus? What happens when you tab away from the “Actions” tab?

**Comment by v-xuanzh** [comment]
Hi @craxal ,We expect the first action like 'Open in Portal' has been focused when clicking the Actions tab.
![image](https://user-images.githubusercontent.com/34729022/62176713-1a948f80-b375-11e9-8920-9905363ec056.png)

**Comment by craxal** [comment]
@v-xuanzh I verified that the first element does in fact have focus when clicking the Actions tab. It's just not visible. If you use the keyboard, then the orange border shows.

Since we're planning on dropping the Actions tab, and this isn't blocking any functionality, I'm going to close as Won't Fix.
